en/micro-integrator/docs/develop/creating-artifacts/using_docker_secrets.md

```diff
@@ -79,36 +79,63 @@ Specify values for the following three parameters:
 ```
 
 ## Enabling secrets in the environment
 
-Once the secrets are added to the environment, you need to enable <b>secure vault</b> in the environment. In a <b>Docker environment</b> you don't need to manually run the Cipher tool. Follow the steps given below.
+Once the secrets are added to the environment, you need to enable <b>secure vault</b> in the environment. In a <b>Docker environment</b>, you don't need to manually run the Cipher tool. Follow the steps given below.
 
 1. Open your Integration Project in WSO2 Integration Studio, which contains all the integration artifacts and the Docker Exporter.
-2. Open the `pom.xml` of the Docker Exporter module and select the <b>Enable Cipher Tool</b> check box as show below.
+2. Open the `pom.xml` of the Docker Exporter module and select the <b>Enable Cipher Tool</b> check box as shown below.
 
     <img src="../../../assets/img/enable-cipher-tool-in-docker.png">
 
-3.  When you build the Docker image from your Docker exporter, the secrets will get enabled in the environment.
+3.  When you build the Docker image from your Docker Exporter, the secrets will get enabled in the environment.
+
+!!! Tip
+     For Docker secrets to be effective, you can create a docker service that includes the image you created in the above step. This can be done by creating a `docker-compose.yml` file and deploying it to the docker swarm. 
+     
+     Given below is a sample `docker-compose.yml` file that can be used to create a simple docker service orchestration.
+     ```bash
+     version: '3.3'
+     services:
+       wso2mi:
+         image: wso2/wso2mi:latest
+         ports:
+           - 8290:8290
+           - 8253:8253
+         secrets:                   
+           - testdockersecret
+     secrets:                       
+       testdockersecret:
+         external: true
+     ```
+     
+     Upon creating the `docker-compose.yml` file, you can deploy the services using the `docker stack deploy` command as follows:
+ 
+     ```bash
+     docker stack deploy -c <PATH_TO_DOCKER_COMPOSE_FILE> <STACK_NAME>
+     ```
+
+     See the [Docker guide](https://docs.docker.com/engine/swarm/secrets/#defining-and-using-secrets-in-compose-files) for more information on defining and using Docker secrets.
 
 ## Configuring the secrets' location
-The Docker secrets exist in default directory locations in the container based on the operating system. Therefore, by default, the server will search for aliases in these directories. The default location for the file secret would be `<MI-HOME>/` directory.
+The Docker and file secrets can be found in default directory locations in the container. The Docker secrets default to `/run/secrets/<secret_name>` in Linux containers, or `C:\ProgramData\Docker\secrets` in Windows containers whereas the default location for the file secret is the `<MI-HOME>/` directory. Therefore, by default, the server will search for aliases in these directories. 
```

Comment:
The Docker secrets and file secrets are stored in default locations in the container environment. The Docker secrets can be found in the following location: 
- **On Linux**: `/run/secrets/<secret_name>`
- **On Windows**: `C:\ProgramData\Docker\secrets`

The default location for the file secret is the `<MI-HOME>/` directory. Therefore, by default, the server will search for aliases in these directories.